docs/src/architecture/secureboot.md

```diff
@@ -0,0 +1,50 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Secure Boot
+
+This section describes secure boot and how to create secure keys.
+
+## Process of Enabling
+
+Secure Boot can be enabled on NixOS using the project Lanzaboote. Secure Boot is a UEFI feature that only allows trusted operating systems to boot.
+Lanzaboote has two components: lzbt and stub. lzbt signs and installs the boot files on the ESP. stub is a UEFI application that loads the kernel and initrd from the ESP.
+
+### Process of Secure keys creation
+
+To create Secure Boot keys, we use sbctl, a popular Secure Boot Key Manager. sbctl is available in Nixpkgs as pkgs.sbctl.
+
+Once you have installed sbctl (or entered a Nix shell), creating your Secure Boot keys requires this command:
```

Comment:
Replace with:

After you installed sbctl or entered a Nix shell, use the following command to create your Secure Boot keys: